qualtran/_infra/gate_with_registers.py

```diff
@@ -157,12 +157,15 @@ def decompose_bloq(self) -> 'CompositeBloq':
             A CompositeBloq containing the decomposition of this Bloq.
 
         Raises:
-            NotImplementedError: If there is no decomposition defined; namely: if
+            DecomposeNotImplementedError: If there is no decomposition defined; namely: if
```

Comment:
the rest of this docstring is no longer accurate

Reply:
updated